.github/workflows/publish.yaml

```diff
@@ -0,0 +1,32 @@
+name: Run forecast
+
+on:
+  schedule:
+    - cron: "0 9 * * *"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout repo content
+        uses: actions/checkout@v2
+
+      - name: setup python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+
+      - name: install python packages
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+          pip install -e .
+
+      - name: upload forecast to Hugging Face
+        env:
+          HF_TOKEN: ${{ secrets.HF_TOKEN }}
```

Comment:
Do you mind changing this to HF_TOKEN_PUSH, just then it doesnt interfere with the other HF_TOKEN we use for running the tests